spec/features/admin/councils/new_spec.rb

```diff
@@ -66,22 +66,23 @@
       # Only Email filled in
       it do
         fill_in 'Email', with: 'C1@council.co.nz'
-        click_button 'Save'
+        expect { click_button 'Save' }.to change { Council.count }.by(0)
         expect(page).to have_text('2 errors prohibited this data from being saved:')
         expect(page).to have_text("Name can't be blank")
         expect(page).to have_text("Short name can't be blank")
-        # expect(click_button('Save')).to change { Council.count }.by(1)
       end
 
-      # # Duplicate Name & Short name
-      # it do
-      #   fill_in 'Name', with: 'New Council 1'
-      #   fill_in 'Short name', with: 'New C1'
-      #   click_button 'Save'
-      #   expect(page).to have_text('2 errors prohibited this data from being saved:')
-      #   expect(page).to have_text('Name has already been taken')
-      #   expect(page).to have_text('Short name has already been taken')
-      # end
+      # Duplicate Name & Short name
+      it do
+        FactoryBot.create :council, name: 'New Council 1', short_name: 'New C1'
+        fill_in 'Name', with: 'New Council 1'
+        fill_in 'Short name', with: 'New C1'
+        fill_in 'Email', with: 'C1@council.co.nz'
+        expect { click_button 'Save' }.to change { Council.count }.by(0)
```

Comment:
```suggestion
        expect { click_button 'Save' }.not_to change { Council.count }
```

Reply:
Rubocop throws an error when that is implemented :frowning_face: